assets/javascripts/bundle.e1a181d9.min.js.map

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@elBoberido I suppose you copied all the new files from the folder `v2.0.5`. How will we maintain this in the future? The style of MkDocs is often updated and then the style of the landing page might differ from the rest of the page. I think you need to update the [script](https://github.com/eclipse-iceoryx/iceoryx/blob/master/tools/website/export-docu-to-website.sh) for the website generation e.g. copying the latest style and javascript from the latest version.

Reply:
Isn't this the same Problem with all the other copies in v1.0.0, v1.0.1, etc?

Reply:
Yeah, you are right it's similar. I guess we have to manually care about it :dizzy_face: